espnet/asr/pytorch_backend/asr.py

```diff
@@ -235,8 +224,8 @@ def __init__(self, subsampling_factor=1, dtype=torch.float32):
         self.ignore_id = -1
         self.dtype = dtype
 
-    def __call__(self, batch, device):
-        """Transforms a batch and send it to a device.
+    def __call__(self, batch, device=None):
```

Comment:
I think this is not your fault but I'm confusing about this `device` and `self.device`. Why this class has two devices? Is it possible to use only one?